modules/sno-du-enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-enabling-workload-partitioning_{context}"]
+= Enabling workload partitioning
```

Comment:
Telcodocs-299 has a dedicated page for how to enable workload partitioning. This section should at least make reference to that section for additional information.